src/main/java/org/launchcode/techjobs/oo/test/JobTest.java

```diff
@@ -1,12 +1,155 @@
+        assertEquals(1, job1.getId());
+        assertEquals(2, job2.getId());
+    }
+/*
+    @Test
+    public void idFieldsDifferByOne(){ assertTrue(job2.getId()-job1.getId()==1);}
+
+    @Test
+    public void nameFieldSet(){assertTrue(test_job1.getName()=="Product tester");}
+
+    @Test
+    public void employerFieldSet(){assertTrue(test_job1.getEmployer() instanceof Employer);}
+
+    @Test
+    public void locationFieldSet(){assertTrue(test_job1.getLocation() instanceof Location);}
+
+    @Test
+    public void testPositionTypeEqualsMethod(){
+        assertTrue(test_job1.getPositionType() instanceof PositionType);
+        assertEquals("Quality control", test_job1.getPositionType().getValue());
+    }
+
+    @Test
+    public void  coreCompetencySet(){
+        assertTrue(test_job1.getCoreCompetency() instanceof CoreCompetency);
+        assertEquals("Persistence", test_job1.getCoreCompetency().getValue());
+    }
+
+ */
+
+    @Test
+    public void testJobsForEquality(){
+        test_job2 = new Job("Product tester", new Employer("ACME"), new Location("Desert"), new PositionType("Quality control"), new CoreCompetency("Persistence"));
+        test_job3 = new Job("Product tester", new Employer("ACME"), new Location("Desert"), new PositionType("Quality control"), new CoreCompetency("Persistence"));
+        assertFalse(test_job2.equals(test_job3));
+    }
+/*
+    @Test
+    public void testToStringContainsCorrectLabelsAndData(){
+        test_job2 = new Job("Product tester", new Employer("ACME"), new Location("Desert"), new PositionType("Quality control"), new CoreCompetency("Persistence"));
+        assertEquals("\n\n",test_job2.toString() );
+    }
+    */
+@Test
+public void testToStringContainsCorrectLabelsAndData() {
+    Job testJob4 = new Job("Product Tester", new Employer("ACME"), new Location("Desert"), new PositionType("Quality control"), new CoreCompetency("Persistence"));
+    assertEquals("\nID: " + testJob4.getId() + "\nName: Product Tester\nEmployer: ACME\nLocation: Desert\nPosition Type: Quality control\nCore Competency: Persistence\n" , testJob4.toString());
+}
+
+    /*
+    @Test
+    public void testToStringStartsAndEndsWithNewLine(){
+        assertEquals("\n" + "ID: 3\n" +
+                "Name: Product tester\n" +
+                "Employer: ACME\n" +
+                "Location: Desert\n" +
+                "Position Type: Quality control\n" +
+                "Core Competency: Persistence\n",test_job1.toString());
+    }
+
+     */
+
+    @Test
+    public void testToStringHandlesEmptyField(){
+        Job test_job4 = new Job("Product Tester", new Employer(""), new Location("Desert"), new PositionType("Quality control"), new CoreCompetency(""));
+        assertEquals("\n" + "ID: 5\n" +
+                "Name: Data not available\n" +
+                "Employer: Data not available\n" +
+                "Location: Data not available\n" +
+                "Position Type: Data not available\n" +
```

Comment:
This one surprises me as passing, since you're passing a Location and Position Type - I wouldn't expect 'Data not available' to show for these two lines. I would expect it for CoreCompetency and Employer though.